<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_new</class>
 <widget class="QDialog" name="create_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>481</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>464</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Create New Crossword</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string comment="General settings for new crosswords.">General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Crossword Type Information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="KComboBox" name="crosswordType">
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="duplicatesEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="CrosswordTypeWidget" name="typeInfoWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="grpDocumentInformation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Document &amp;Information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="1" colspan="5">
           <widget class="KLineEdit" name="notes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="clickMessage">
             <string>Notes about the crossword</string>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="5">
           <widget class="KLineEdit" name="author">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="clickMessage">
             <string>The author(s) of the crossword</string>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="5">
           <widget class="KLineEdit" name="title">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="clickMessage">
             <string>The title of the crossword</string>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblNotes">
            <property name="text">
             <string>&amp;Notes:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>notes</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblTitle_2">
            <property name="text">
             <string>&amp;Title:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>title</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblCopyright">
            <property name="text">
             <string>&amp;Copyright:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>copyright</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblSize">
            <property name="text">
             <string>Crossword &amp;Size:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>columns</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="KIntNumInput" name="columns">
            <property name="value">
             <number>15</number>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="referencePoint">
             <number>15</number>
            </property>
            <property name="suffix">
             <string> columns</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string notr="true">x</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="KIntNumInput" name="rows">
            <property name="value">
             <number>15</number>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="referencePoint">
             <number>15</number>
            </property>
            <property name="suffix">
             <string> rows</string>
            </property>
            <property name="sliderEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblAuthor">
            <property name="text">
             <string>&amp;Author:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>author</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="5">
           <widget class="KLineEdit" name="copyright">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="clickMessage">
             <string>Copyright information about the crossword</string>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTemplate">
      <attribute name="title">
       <string>Template</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QCheckBox" name="useTemplate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblTemplateLocation">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Location:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>templateLocation</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="templateLocation">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QSplitter" name="splitter">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="lblTemplateSearchLine">
             <property name="text">
              <string>S&amp;earch:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>templateSearchLine</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTreeView" name="templates">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="animated">
              <bool>true</bool>
             </property>
             <property name="allColumnsShowFocus">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KLineEdit" name="templateSearchLine">
             <property name="showClearButton" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_2">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="lblPreview">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Preview:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring></cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="CrosswordPreviewWidget" name="preview">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>256</width>
               <height>256</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>256</width>
               <height>256</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblTemplateType">
             <property name="text">
              <string>Crossword Type:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>templateLocation</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lblTemplateSize">
             <property name="text">
              <string>Size:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>templateLocation</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lblTemplateNotes">
             <property name="text">
              <string>Notes:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
             <property name="buddy">
              <cstring>templateLocation</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="templateType">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="templateSize">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="templateNotes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblUseTemplate">
         <property name="text">
          <string>&amp;Use Template:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>useTemplate</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>CrosswordTypeWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/crosswordtypewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CrosswordPreviewWidget</class>
   <extends>QLabel</extends>
   <header>dialogs/crosswordpreviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>crosswordType</tabstop>
  <tabstop>title</tabstop>
  <tabstop>author</tabstop>
  <tabstop>copyright</tabstop>
  <tabstop>notes</tabstop>
  <tabstop>useTemplate</tabstop>
  <tabstop>templateLocation</tabstop>
  <tabstop>templateSearchLine</tabstop>
  <tabstop>templates</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>useTemplate</sender>
   <signal>toggled(bool)</signal>
   <receiver>templateLocation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTemplate</sender>
   <signal>toggled(bool)</signal>
   <receiver>splitter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTemplate</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblTemplateLocation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>create_new</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>create_new</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
